vip-helpers/vip-utils.php

```diff
@@ -1058,7 +1058,13 @@ function wpcom_vip_load_plugin( $plugin = false, $folder = false, $load_release_
 
 		return _wpcom_vip_include_plugin( $includepath );
 	} else {
-		$error_msg = sprintf( 'wpcom_vip_load_plugin: Unable to load plugin `%s`; the path `%s` does not exist.', $plugin, $includepath );
+		$error_msg = sprintf( 'wpcom_vip_load_plugin: Unable to load plugin `%s`', $plugin );
+		if ( $includepath ) {
```

Comment:
If the variable is not defined, this will throw an error.
Or, is it always defined but sometimes empty?

Reply:
Defined as `null` here: https://github.com/Automattic/vip-go-mu-plugins/pull/923/files#diff-fb296e3bc7c2a02335f61196cce2f90fR1026